### DISCUSSION TITLE
Alerting: allow disabling grafana alerting while still showing page with alerting rules

### DISCUSSION THREAD

**Discussion by jmichalek132** [grafana/grafana#71695] [Alerting]
Hi, we have we use the LGMT stack, we want to use only Mimir & Loki ruler for alerting, because we have a gitops pipeline in place and want to have a pipeline for validating alerts / recording rules & code reviews. Because of this, we want to disable the creation of alerts within the grafana itself as those alerts would bypass this process. However, right now the only way to view what alerts have been deployed to mimir / loki ruler is via grafana, but disabling the grafana alerting feature disables this page too.
 
So I would like to request a separate configuration option for disabling alerting in grafana itself while still showing the page with alerts.

I know currently we could at least disable the execution of the alerts in Grafana, but given people would end up creating alerts thinking they have alerting now in place, this is less than ideal.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82615. Feel free to continue the discussion around this there. Thank you!
